k9mail/src/main/java/com/fsck/k9/notification/DeviceNotifications.java

```diff
@@ -42,6 +43,46 @@ public static DeviceNotifications newInstance(NotificationController controller,
         return new DeviceNotifications(controller, actionCreator, lockScreenNotification, wearNotifications);
     }
 
+    public Notification buildUrgentNotification(Account account, NotificationData notificationData) {
+        int unreadMessageCount = notificationData.getUnreadMessageCount();
```

Comment:
method seems ok to me, but I'm not involved in the notifications much. ping @cketti please review